config.ini.sample

```diff
@@ -38,6 +38,8 @@ summary_file=output/wer_summary.json
 word_accuracy_file=output/wer_word_accuracy.csv
 ;JSON file with STT output
 stt_transcriptions_file=output/stt_transcriptions.csv
+;Directory where sclite is installed if sclite is to be used for Analysis
+;sclite_directory=
```

Comment:
You may want to give a pointer here to `https://github.com/usnistgov/SCTK#sctk-basic-installation` or some guide about where  this is typically installed.
My first thought was someone would clone the SCTK repo directly in our root working dir (or a sub-dir), if that is really the best idea then it would need a .gitignore.
But I suspect you're supposed to do something like /usr/bin/SCTK.

Reply:
Added the link to the install. Where `sclite` is installed was intentionally left up to the user through the addition of this variable. Personally I don't want to have it on my `PATH` and so just install it into an applications directory and supply that path into the config. 

There is potential room for improvement, though I'd suggest in follow on PRs, to include a script for installing `sclite` at which point we should probably make a subfolder and isolate `sclite` resources to it.